### PR TITLE
Mobile form

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- [ ]  Add in all fields placeholder text.
- [ ]  Add in all fields the correct type.
- [ ]  Add a Name field that accepts up to 30 chars.
- [ ]  Add an Email field that accepts only emails.
- [ ]  Add a message field that accepts up to 500 chars.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
+                <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                    <form action="#" method="post">
```
Comment (huseyinbiyik) [inline index.html]: Please sign up for the FormSpree service and add it here.  So you can get emails from your contact form. This line needs to look similar to the line below.
```<form action="https://formspree.io/f/{form_id}" method="post">```

File: index.html [hunk 2]
```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
```
Comment (huseyinbiyik) [inline index.html]: Kindly stick to the design and fix the left top gray area. Check the images below.
Figma template:
Screenshot 2023-01-18 at 12 38 56 AM
Screenshot 2023-01-18 at 12 39 23 AM

File: index.html [hunk 3]
```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
...
+                        <fieldset class="mobile-contact">
+                            <label for="username">
+                                <input type="text" class="input" required name="username" maxlength="30" placeholder="Username">
+                            </label>
+                            <label for="email">
+                                <input type="email" class="input" required name="email" placeholder="name@domain.com">
+                            </label>
+                            <label for="message">
+                                <textarea name="message" class="message-mobile" required maxlength="500" placeholder="Write your message here"></textarea>
+                            </label>
```
Comment (huseyinbiyik) [inline index.html]: Please stick to the design and add right amount of padding for each side of inputs. Here is the difference.
Screenshot 2023-01-18 at 12 40 41 AM
Screenshot 2023-01-18 at 12 41 14 AM

Comment (huseyinbiyik) [inline index.html]: Please fix the font issues here. Kindly try to follow the Figma template. 👍
Screenshot 2023-01-18 at 12 42 13 AM
Screenshot 2023-01-18 at 12 42 26 AM

File: index.html [hunk 4]
```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
...
+                            <div class="button-contact">
+                              <button type="submit" class="btn last-btn">Get in touch</button>
```
Comment (huseyinbiyik) [inline index.html]: Kindly add hover and pressed state for your submit button as mentioned on the project requirements.
Screenshot 2023-01-18 at 12 56 26 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Moses-chibuike and @Kaghenimbale,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Collaborative work 🧠🧠
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
- You have some overflowing issues right now. Kindly check the video below. Please try to fix them to have full responsive web site.
Screen Recording 2023-01-18 at 12 46 02 AM

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Moses-chibuike) [reply index.html:184]: Please 🙏  can you explain more? What are the font issues?

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Moses-chibuike ans @Kaghenimbale 

I am Favour 👩🏽 , your reviewer for this PR.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Moses-chibuike) [reply index.html]: Thanks for the review @huseyinbiyik

File: index.html (hunk 2 above)
Comment (Moses-chibuike) [reply index.html]: Thanks for the review @huseyinbiyik

File: index.html (hunk 3 above)
Comment (Moses-chibuike) [reply index.html]: Thanks for the review @huseyinbiyik

Comment (Moses-chibuike) [reply index.html]: Thanks for the review @huseyinbiyik

File: index.html (hunk 4 above)
Comment (Moses-chibuike) [reply index.html]: Thanks for the review @huseyinbiyik

**Comment by Moses-chibuike** [comment]
Thanks for approving @Favourezeugwa

**Pull request merged**